torch/testing/_internal/common_nn.py

```diff
@@ -5043,8 +5043,12 @@ def _do_test(self, test_case, module, input):
             # could probably unify check_jacobian above with this.
             params = tuple(x for x in module.parameters())
             num_inputs = len(input_tuple)
+            # These modules don't actually take the arguments as inputs so forward check will fail for these
+            # We will be able to re-enable this when https://github.com/pytorch/pytorch/issues/49171 is done
+            check_forward = len(params) == 0
```

Comment:
nit: I had to stop here and think for a few seconds. The question that came to my mind was ,"when are there no parameters?" It would be nice to clarify in a comment that here we are checking forward mode AD for nn.Modules with no parameters like nn.Softmax, nn.Relu, ...